tests/foreman/api/test_template_combination.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""Tests for template combination
+
+@Requirement: TemplateCombination
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: API
+
+@TestType: Functional
+
+@CaseImportance: Medium
+
+@Upstream: No
+"""
+from nailgun import entities
+from requests.exceptions import HTTPError
+from robottelo.decorators import tier1, skip_if_bug_open
```

Comment:
not sure why do you look on that line, I highlighted `from robottelo.decorators import tier1, skip_if_bug_open`